config/locales/en.yml

```diff
@@ -1611,8 +1611,8 @@ en:
       hi: "Hi %{to_user},"
       greeting: "Hi,"
       commented:
-        subject_own: "[OpenStreetMap] %{commenter} has commented on one of your changesets"
-        subject_other: "[OpenStreetMap] %{commenter} has commented on a changeset you are interested in"
+        subject_own: "[OpenStreetMap] %{commenter} has commented on your changeset %{changeset_id}"
+        subject_other: "[OpenStreetMap] %{commenter} has commented on changeset %{changeset_id} you are interested in"
         your_changeset: "%{commenter} left a comment at %{time} on one of your changesets"
```

Comment:
Unfortunately, for technical reasons explained below, you can't change the interpolation variables in a translation, without also changing the key.

The technical reason is that the interpolation variables must be exactly the same in all translations for the same key. It can take several weeks, or longer, for all the translators to update their translations, and in the meantime the code would not work in those languages. So if you change the interpolations in a value, you need to change the key (e.g. `subject_own`) to something else too.